ext/date/tests/createFromTimestamp.phpt

```diff
@@ -286,3 +227,4 @@ MyDateTimeImmutable::createFromTimestamp(0): object(MyDateTimeImmutable)#%d (3)
   ["timezone"]=>
   string(6) "+00:00"
 }
+
```

Comment:
Please remove trailing newlines. We use a single newline on on the last non-blank line.